tests/__mocks__/handlers.ts

```diff
@@ -2,6 +2,7 @@ import { http, HttpResponse } from "msw";
 import { db } from "./db";
 import issueTemplate from "./issue-template";
 import { Label } from "../../src/types/github";
+import { ReadableStream } from "node:stream/web";
```

Comment:
@whilefoo what is this for?

Reply:
It's used as a type in the function below. Typescript was complaining about it so I imported it